### ISSUE TITLE
Uncaught TypeError: L.Application is not a constructor

### ISSUE THREAD

**Issue by mr-akashdesai** [splinetool/react-spline#65]
Hi, I'm having some issues rendering my [spline](https://my.spline.design/fantomlogo-fabfaa935f3579162f29ca577173ecac/) onto my project. It all looks fine on the CodeSanbox link provided but when using it with my own app there seems to be an issue. I am using React w/ Typescript and webpack. Here is the component:

<img width="708" alt="image" src="https://user-images.githubusercontent.com/43957034/180654227-ca892a76-3238-4da8-9699-af1911fb6837.png">

Whenever I try to render the component, a TypeError is thrown regarding `L.Application is not a constructor`: 

<img width="594" alt="image" src="https://user-images.githubusercontent.com/43957034/180653841-7aa70f09-08b8-441d-aa02-28e5ae849ad1.png">

I'm not really sure what the error means but it was addressing a `ForwardRef` component so it could potentially be something around the `ForwardRefExocticComponent` as shown below 

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/43957034/180653938-950a3697-e4bd-495e-af6c-9f6668b684ed.png">

Please let me know if you have any ideas. Thanks 🙂

**Comment by marcofugaro** [comment]
> I am using React w/ Typescript and webpack.

I think the issue is in your setup. Could you share an example repo of the problem happening?

**Comment by mr-akashdesai** [comment]
Yeah sure, I've just made a new repo, partially cloning my app but using all the same dependencies and config. 
If you want to run it:
`npm install`
then
`npm start`

https://github.com/mr-akashdesai/spline-mockup

**Comment by marcofugaro** [comment]
Your repo starts successfully on my machine, it's a problem with your local dependencies. You should delete the node_modules and reinstall locally.

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/7217420/180820196-d2fbdd87-330d-4a07-bf56-170672175940.png">

**Comment by mr-akashdesai** [comment]
Were you able to see the spline on the rendered component? The error is thrown within the developer tools not in the terminal since it's not a syntax error. I created a new repo and reinstalled all the node modules, so I don't think that is the issue.

**Comment by mr-akashdesai** [comment]
Reinstalled node_modules again, same issue still. 
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/43957034/180823601-e8c9e660-fd3c-405f-8dd3-b39ca1724a20.png">
